### PR TITLE
CANVAS-64 - Set up git-secrets

### PR DESCRIPTION
**Pull request by chern**

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -16,19 +16,21 @@ DEBUG='provider:*,registrar'
 REG_WS_API_URL=https://webservicesqa.it.ucla.edu
 REG_WS_CERT_NAME=/replace/with/pathto/file.cer
 REG_WS_PRIVATE_KEY=/replace/with/pathto/file.key
-REG_WS_CLIENT_ID=
-REG_WS_SECRET=
+SECRET_REG_WS_CLIENT_ID=
+SECRET_REG_WS_SECRET=
```
Comment (rlorenzo) [inline .env.dist:20]: @chern Strange to have SECRET_*_SECRET

Just rename it to SECRET_REG_WS_PASSWORD

File: .env.dist [hunk 2]
```diff
@@ -16,19 +16,21 @@ DEBUG='provider:*,registrar'
...
-VIDEORES_FTP_HOST=
-VIDEORES_FTP_USERNAME=
-VIDEORES_FTP_PWD=
-VIDEORES_FILEPATH=
\ No newline at end of file
+SECRET_VIDEORES_FTP_HOST=
+SECRET_VIDEORES_FTP_USERNAME=
+SECRET_VIDEORES_FTP_PWD=
+SECRET_VIDEORES_FILEPATH=
+SECRET_VIDEORES_TOKEN_SECRET=
```
Comment (rlorenzo) [inline .env.dist:35]: @chern Rename to just SECRET_VIDEORES_TOKEN

File: .env.dist [hunk 3]
```diff
@@ -16,19 +16,21 @@ DEBUG='provider:*,registrar'
 REG_WS_API_URL=https://webservicesqa.it.ucla.edu
```
Comment (rlorenzo) [inline .env.dist:16]: @chern What about the database user/password? Also PLATFORM_CLIENTID?

File: .env.dist [hunk 4]
```diff
@@ -16,19 +16,21 @@ DEBUG='provider:*,registrar'
...
 REG_WS_CERT_NAME=/replace/with/pathto/file.cer
 REG_WS_PRIVATE_KEY=/replace/with/pathto/file.key
-REG_WS_CLIENT_ID=
-REG_WS_SECRET=
+SECRET_REG_WS_CLIENT_ID=
+SECRET_REG_WS_SECRET=
 # Bruincast configs
 HOST=bclive.oid.ucla.edu
 SECRET=******
```
Comment (rlorenzo) [inline .env.dist:23]: @chern This name is now too generic.

File: .env.dist [hunk 5]
```diff
@@ -16,19 +16,21 @@ DEBUG='provider:*,registrar'
...
 TOKEN_NAME=wowzatoken
 VALIDITY=90000
-BRUINCAST_API_LOGIN=
-BRUINCAST_API_USERNAME=
-BRUINCAST_API_PASSWORD=
-BRUINCAST_API=
+SECRET_BRUINCAST_API_LOGIN=
```
Comment (rlorenzo) [inline .env.dist:26]: @chern This is just the URL, right? Doesn't need to be secret

File: .env.dist [hunk 6]
```diff
@@ -16,19 +16,21 @@ DEBUG='provider:*,registrar'
...
+SECRET_BRUINCAST_API_USERNAME=
+SECRET_BRUINCAST_API_PASSWORD=
+SECRET_BRUINCAST_API=
```
Comment (rlorenzo) [inline .env.dist:29]: @chern Same here

**Comment by maxpluez** [comment]
Code reviewed and tested. Git secret prevented a commit.
Screenshot from 2020-09-09 17-51-59

**Pull request merged**